ggdeploymentd/src/deployment_handler.c

```diff
@@ -1577,7 +1577,11 @@ static GglError resolve_dependencies(
 
         if (!found_local_candidate) {
             // Resolve with cloud and download recipe
-            static uint8_t resolve_component_candidates_response_buf[16384]
+            static uint8_t resolve_component_candidates_response_buf
+                [16384] // TODO: Clean up usages of static variables in this
+                        // file, many may be more appropriately non-static to
+                        // prevent issues where the memory is not re-initalized
+                        // to zero when the function is re-called
```

Comment:
If memory needs to be zeroed should be memset.
Though this might be fine as non-static, yeah.